### DISCUSSION TITLE
Add horn activation and additional sensors?

### DISCUSSION THREAD

**Discussion by repd79** [TobiasLaross/Hass-Lynk-Co#32] [Ideas]
Hi, I've been loving the Lynk & Co integration with Home Assistant and it is great, thank you! I specially use the AC, as it's very hot here.  However, it would be fantastic if you could include:
- **Horn Activation**: This feature would be incredibly useful for those times when we can't locate the car in a busy parking lot during the day. It could also serve as an additional alert if the home security system detects a potential intruder.
- **Additional Sensors**: Integrating more sensors from the car such as windows, roof, tire pressure, and other diagnostic information would be amazing. Having these sensors available in HA would allow us to create great dashboards.

Additionally, I'm curious about the feasibility of creating a standalone alternative Lynk & Co app that incorporates these features. It would be great to have a dedicated app so we don’t entirely rely on Home Assistant for all these functionalities.

Thanks

**Comment by TobiasLaross** [comment]
Created 2 new issues to track your requests. 
regarding the feasibility of creating a standalone app it could be done but not something I will implement. But I use native iOS app with hass as backend, you can create a fork of https://github.com/TobiasLaross/IntelliNest and adapt for your setup

**Reply by repd79** [reply]
Thanks a lot! I actually need it for Android, but I will explore this approach. Appreciate the guidance!

**Reply by TobiasLaross** [reply]
You can test the horn functionality in 1.5.0, I have not actually tested the sound at all. Also does it make sense to have a stop_honk?

**Comment by TobiasLaross** [comment]
The new sensors are added in 1.6.0

**Comment by repd79** [comment]
Hi, this was super fast, thank you! Honk works great. It sounds three times and you mostly have no time to stop it.

I will share a mobile dashboard when it's ready in case it's useful for someone. 

Thanks again!
